ledger-core/insolar/payload/validation.go

```diff
@@ -177,3 +179,44 @@ func (m *VStateReport) validateUnimplemented() error {
 
 	return nil
 }
+
+func (m *VFindCallRequest) Validate(currPulse PulseNumber) error {
+	var (
+		lookAtPulse   = m.GetLookAt()
+		calleePulse   = m.GetCallee().GetLocal().Pulse()
```

Comment:
please, add check that callee is valid SelfRef (`IsSelfScope`), for now. That check will be replaced with `IsObjectReference` in the future

Reply:
sure, not problem)

Reply:
Fixed